tests/test_loss.py

```diff
@@ -105,3 +105,37 @@ def test_bmn_loss():
     assert_array_almost_equal(output_loss[1].numpy(), output_tem_loss)
     assert_array_almost_equal(output_loss[2].numpy(), output_pem_reg_loss)
     assert_array_almost_equal(output_loss[3].numpy(), output_pem_cls_loss)
+
+
+def test_ssn_loss():
```

Comment:
Besides the sub function of the `SSNLoss`, you must also test the whole `SSNLoss` foward since it is the common case.